DataFormats/TauReco/src/classes_def_2.xml

```diff
@@ -1,6 +1,18 @@
 <lcgdict>
   
-  <class name="reco::PFTau" ClassVersion="19">
+  <class name="reco::PFTau" ClassVersion="21">
+   <version ClassVersion="21" checksum="759808483"/>
+     <field name="leadPFChargedHadrCand_" transient="true"/>
+     <field name="leadPFNeutralCand_" transient="true"/>
+     <field name="leadPFCand_" transient="true"/>
```

Comment:
all transients of type AtomicPtrCache should have an explicit reset in the ioread rules in this file.